test/single_file/new_expr_array_size.cc

```diff
@@ -0,0 +1,8 @@
+class foo {
+public:
+  foo(int x) {}
+};
+
+int main() { 
+  new foo[2]{4, 5}; 
+}
```

Comment:
Please there is a new line at end of file.

Reply:
Fixed on [52f1fa6](https://github.com/mc-imperial/dredd/pull/272/commits/52f1fa68a33d9b0425c044b449ff5b4f7dbbae90)